cmsp-post (#2)
selfsk/selfsk.github.io: _posts/2021-03-18-how-i-didnt-become-an-instructor.md

---
layout: post
title: "How I didn't become a CMSP instructor"
date: 2021-03-18 6:30pm -0700
categories: motorcycles education learning
---

Recently I've attempted (yes, attempted) to get a [CMSP](https://cmsp.msi5.com/) (California Motorcyclist Safety Program) certification, to get a better understanding how people are getting prepared to ride a motorcycle.

> I didn't go through, mostly because what I've seen and heard so far just make me worry about rider's safety and my internal conflict person just can not let it slide.

Reviewer: ```suggestion
I didn't go through, mostly because what I've seen and heard so far just makes me worry about rider's safety and my internal conflict person just can not let it slide.
```

## My background

> I've been riding for 5 years so far, not much in terms of time, but in that time I was able to put about 100k miles, doing various things - from commuting to participate in racing. 

Reviewer: ```suggestion
I've been riding for 5 years so far, not much in terms of time, but in that time I was able to put about 100k miles, doing various things - from commuting to participation in racing. 
```

> I never did a basic riding course, and was learning things on my own from books and videos available (semi)publicly on internet (see [Links](#Links) at the bottom of post). After I get more experienced riding motorcycle I enrolled in IRC (Intermediate Riding Course), which I finished without too much issues.

Reviewer: ```suggestion
I never took a basic riding course, and was learning things on my own from books and videos available (semi)publicly on the internet (see [Links](#Links) at the bottom of the post). After I got more experienced rider I enrolled in IRC (Intermediate Riding Course), which I completed without too much hassle.
```

> I did my first trackday shortly after finishing IRC, at Thunderhill East, with Z2 which was an awesome experience which really showed what bike is capable of, and how many I need to learn to become really proficient rider.

Reviewer: ```suggestion
I did my first trackday shortly after finishing IRC, at Thunderhill East with Z2. It was an awesome event which showed what my bike is capable of, and how much I need to learn to become a proficient rider.
```

> Fast forward to 2020, I did participate in AFM racing, only 1 round so far, didn't win anything but, again, was able to learn a lot about capabilities of my motorcycle (CBR600RR '08) and myself. In between of those dates I took an extra Ken Hill's riding school provided by [Carters@TheTrack](https://cartersatthetrack.com/) and practice with mini-bikes on kart tracks.

Reviewer: ```suggestion
Fast forward to 2020, I participated in AFM racing, only 1 round so far, didn't win anything but, again, was able to learn a lot about capabilities of my motorcycle (CBR600RR '08) and myself. In between of those dates I took an extra Ken Hill's riding school provided by [Carters@TheTrack](https://cartersatthetrack.com/) and practiced with mini-bikes on kart tracks.
```
